src/k8s-extension/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.4.2
+++++++++++++++++++
+* microsoft.azuremonitor.containers: ContainerInsights Extension Managed Identity Auth Enabled by default
```

Comment:
I would like to confirm if this change will cause a breaking change for the customer?

Reply:
@zhoxing-ms  this is the change similar to the aks https://github.com/Azure/azure-cli/pull/26356. This change is for arc customers so that they when they enable monitoring addon the MSI mode will be enabled by default, from container insights team, this is a not breaking change since we currently already support both modes.